pyproject.toml

```diff
@@ -66,7 +66,7 @@ dependencies = [
 
 [project.optional-dependencies]
 allauth = [
-  "django-allauth>=0.61.1,<0.64.0",
+  "django-allauth>=0.61.1,<64.2.0",
```

Comment:
same holds for allauth, should I bump the minimum version? https://docs.allauth.org/en/latest/release-notes/recent.html

Reply:
ok with me.

Reply:
they have split up the dependencies and our typical use-cases are the `socialaccounts` or OpenID providers, should I add it like this?
ref: https://docs.allauth.org/en/latest/socialaccount/providers/openid.html#openid
```
django-allauth[socialaccount,openid]>=64.1.0,<64.2.0",
```

Reply:
Yes, I don't think we need the others: https://github.com/pennersr/django-allauth/blob/main/setup.cfg#L50 (steam anyone?)

Reply:
`saml` could be interesting, but I haven't tested it so far, it could simplify the shibboleth setup.
`mfa`, multi-factor?

Reply:
yes, `saml` could be useful, but I don't want to enter this business. Feel free to try that with your setup if you find the time. `mfa` is something for the future I guess. People can always install more stuff and costimize rdmo locally.